fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -563,7 +563,11 @@ SIMPLE_COMMENT
     ;
 
 BRACKETED_COMMENT
-    : '/*' ('+'? [ \r\n\t\u3000]* | ~'+' .*?) '*/' -> channel(HIDDEN)
+    : '/*'([ \r\n\t\u3000]* | ~'+' .*?) '*/' -> channel(HIDDEN)
+    ;
+
+OPTIMIZER_HINT
+    : '/*+' .*? '*/' -> channel(2)
```

Comment:
could support all hint(join, agg_stage, agg_skew, cte) in future?

Reply:
Yes, all this hint can be parsed to hintNode.